2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,38 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+
+let emptyBottle = 0;
+let bottleFull = 100;
+let pouringUnits = 10;
+
 let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return this.volume = bottleFull
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    return this.volume += pouringUnits;
```

Comment:
Almost correct! You are missing this requirement: ``Volume will be 100 when bottle is full and 0 when empty.``